tools/eslint-rules/rules-utils.js

```diff
@@ -64,8 +74,8 @@ module.exports.inSkipBlock = function(node) {
 
 function hasSkip(expression) {
   return expression &&
-         expression.callee &&
-         (expression.callee.name === 'skip' ||
-         expression.callee.property &&
-         expression.callee.property.name === 'skip');
+    expression.callee &&
+    (expression.callee.name === 'skip' ||
+      expression.callee.property &&
+      expression.callee.property.name === 'skip');
```

Comment:
Nit: unrelated change. The same applies to the part above.

Reply:
removed.